qt/widgets/common/src/UserSubWindowFactory.cpp

```diff
@@ -127,3 +127,25 @@ QStringList UserSubWindowFactoryImpl::keys() const {
   }
   return key_list;
 }
+
+BaseEncoder *UserSubWindowFactoryImpl::findEncoder(QWidget *window) {
```

Comment:
Can we have some Doxygen documentation here and the method below, please?

Reply:
It might be worth stating in the docs that this has to return a bare pointer so `sip` can understand and wrap it?